src/components/back-link/index.njk

```diff
@@ -0,0 +1,49 @@
+---
+title: Back link
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+
+The back link reuses the same interface and parameters as the [Breadcrumbs](/components/Breadcrumbs) component.
```

Comment:
Small thing, path is broken to breadcrumb component. Should be `/components/breadcrumb`